website/guides/terraform-provider-development-program.html.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "guides"
+page_title: "Terraform Provider Development Program"
+sidebar_current: "guides-terraform-provider-development-program"
+description: This guide is intended for vendors who're interested in having their platform supported by Teraform. The guide walks vendors through the steps involved in creating a provider and applying for it to be included with Terraform.
+---
+
+# Terraform Provider Development Program
+
+The Terraform Provider Development Program allows vendors to build
+Terraform providers that are officially approved and tested by HashiCorp and
+listed on the official Terraform website. The program is intended to be largely
+self-serve, with links to information sources, clearly defined steps, and
+checkpoints.
+
+-> **Building your own provider?** If you're building your own
+provider and aren't interested in having HashiCorp officially maintain
+the provider, refer to the
+[Writing Custom Providers guide](/guides/writing-custom-terraform-providers.html).
+
+## What is a Terraform Provider?
+
+Terraform is used to create, manage, and manipulate infrastructure resources.
+Examples of resources include physical machines, VMs, network switches, containers, etc.
+Almost any infrastructure noun can be represented as a resource in Terraform.
+
+A provider is responsible for understanding API interactions with the underlying
+infrastructure like a cloud (AWS, GCP, Azure), a PaaS service (Heroku), a SaaS
+(service DNSimple, CloudFlare), or on-prem resources (vSphere). It then exposes
+these as resources users can code to. Terraform presently supports more than
+70 providers, a number that has more than doubled in the past 12 months.
+
+All providers integrate into and operate with Terraform exactly the same way.
+The table below is intended to help users understand who develops, maintains
+and tests a particular provider.
+
+![Provider Engagement Table](/assets/images/docs/engage-table.png)
+
+This document is primarily intended for the "HashiCorp/Vendors" row in
+the table above. Community contributors who’re interested in contributing to
+existing providers or building new providers should refer to the
+[Writing Custom Providers guide](/guides/writing-custom-terraform-providers.html).
+
+## Provider Development Process
+
+The provider development process is divided into six steps below. By following
+these steps, providers can be developed alongside HashiCorp to ensure new
+providers are able to be published in Terraform as quickly as possible.
+
+![Provider Development Process](/assets/images/docs/process.png)
```

Comment:
This image contains a "Recruit" starting point, but that isn't mentioned in the guide

Reply:
Good idea. done.

Reply:
I'll let Burzin look at that. I think this doesn't have to block this.

Reply:
Agreed, not a blocker